### ISSUE TITLE
networkx2.4: Graph Object methods removed

### ISSUE THREAD

**Issue by aschmiedeke** [e-koch/FilFinder#54]
When running FilFinder (version 1.6) I got the following AttributeError message:

```
  File "/usr/local/lib/python3.7/dist-packages/fil_finder/filfinder2D.py", line 641, in analyze_skeletons
    max_prune_iter=max_prune_iter)
  File "/usr/local/lib/python3.7/dist-packages/fil_finder/filament.py", line 286, in skeleton_analysis
    if prev_G.node == G[0].node:
AttributeError: 'Graph' object has no attribute 'node'
```

Changing line 286 in filament.py from 
` if prev_G.node == G[0].node:`  
to
` if prev_G.nodes == G[0].nodes:`
solved the problem for me.

See also [https://networkx.github.io/documentation/stable/release/release_2.4.html#deprecations](https://networkx.github.io/documentation/stable/release/release_2.4.html#deprecations)

**Comment by e-koch** [comment]
Thanks! This is fixed in master. I'll push a new release soon, but in the meantime, you can install the fixed version with `pip install git+https://github.com/e-koch/FilFinder.git@master`.

**Comment by aschmiedeke** [comment]
Sorry for the late reply! I installed the fixed version and things are running fine! Thank you!

**Comment by e-koch** [comment]
I've released v1.7 on pip that includes this fix.
